core/pom.xml

```diff
@@ -28,7 +28,8 @@
             -->
             <groupId>org.evomaster</groupId>
             <artifactId>evomaster-client-java-controller</artifactId>
-            <scope>test</scope>
+            <!-- Need this module for runWithDeterministicLogger in LoggingUtil -->
+<!--            <scope>test</scope>-->
```

Comment:
this should be reverted. as `runWithDeterministicLogger` is only used in the tests, move that function in a new class in test scope. so no need to modify pom file